src/group_theory/submonoid/centralizer.lean

```diff
@@ -47,8 +47,8 @@ variables {S}
 @[to_additive] lemma mem_centralizer_iff {z : M} : z ∈ centralizer S ↔ ∀ g ∈ S, g * z = z * g :=
 iff.rfl
 
-@[to_additive] instance decidable_mem_centralizer [decidable_eq M] [fintype M]
-  [decidable_pred (∈ S)] : decidable_pred (∈ centralizer S) :=
+@[to_additive] instance decidable_mem_centralizer [∀ a : M, decidable $ ∀ b ∈ S, b * a = a * b] :
+   decidable_pred (∈ centralizer S) :=
```

Comment:
I think this might make more sense as
```suggestion
@[to_additive] instance decidable_mem_centralizer [decidable_pred (λ a, ∀ b ∈ S, b * a = a * b)] :
   decidable_pred (∈ centralizer S) :=
```
or possibly less successfully:
```suggestion
@[to_additive] instance decidable_mem_centralizer (a) [decidable $ ∀ b ∈ S, b * a = a * b)] :
   decidable (a ∈ centralizer S) :=
```

Reply:
I manually checked if lean still found the original instance with the latter formulation, and it did, so I went with that.